share/picongpu/pypicongpu/template/etc/picongpu/N.cfg.mustache

```diff
@@ -95,6 +95,13 @@ fi
   {{/stop_iteration}}
 {{/moving_window}}
 
+{{#binomial_current_interpolation}}
+TBG_currentInterpolation="--currentInterpolation binomial"
+{{/binomial_current_interpolation}}
+{{^binomial_current_interpolation}}
+TBG_currentInterpolation="--currentInterpolation none"
+{{/binomial_current_interpolation}}
```

Comment:
@ikbuibui suggested to remove this to be less less verbose
@psychocoderHPC what do you think?